src/Libraries/Microsoft.Extensions.Telemetry/Logging/LoggerEnrichmentOptions.cs

```diff
@@ -37,10 +37,24 @@ public class LoggerEnrichmentOptions
     /// <remarks>
     /// Reading available debugging files produces richer stack traces, but can cost a substantial amount of time
     /// to generate. As a result, this option should only be turned on in development scenarios, not for production use.
+    ///
+    /// This property has no effect if <see cref="CaptureStackTraces"/> is <see langword="false"/>.
+    /// 
     /// This defaults to <see langword="false"/>.
     /// </remarks>
     public bool UseFileInfoForStackTraces { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether to include exception messages in stack traces.
+    /// </summary>
+    /// <value>
+    /// This defaults to <see langword="false"/>.
+    /// </value>
+    /// <remarks>
+    /// This property has no effect if <see cref="CaptureStackTraces"/> is <see langword="false"/>.
+    /// </remarks>
+    public bool IncludeExceptionMessageInStackTraces { get; set; }
```

Comment:
True by default?

Reply:
It should be turned off by default to prevent sensitive data leaking